src/backend/distributed/commands/table.c

```diff
@@ -1133,17 +1142,35 @@ ErrorIfUnsupportedAlterTableStmt(AlterTableStmt *alterTableStatement)
 				break;
 			}
 
-			case AT_SetNotNull:
+			case AT_EnableTrig:
+			case AT_EnableAlwaysTrig:
+			case AT_EnableReplicaTrig:
+			case AT_EnableTrigUser:
+			case AT_DisableTrig:
+			case AT_DisableTrigUser:
 			case AT_EnableTrigAll:
 			case AT_DisableTrigAll:
+			{
+				if (!IsCitusLocalTable(relationId))
+				{
+					const char *relationName = alterTableStatement->relation->relname;
+					ereport(ERROR, (errmsg("cannot execute ALTER TABLE ENABLE/DISABLE "
```

Comment:
> we can consider such a generic error message.

That makes sense to me

> errhint: Use citus local tables for triggers

However, I can't get the benefit in telling that triggers are supported for citus local tables, I mean, making a postgres table a citus local table instead of a distributed table doesn't seem to be an alternative to me, as a user

I can vote for:
> errmsg: Cannot execute the command on table "%s" because triggers are not supported for distributed and reference tables